include/swift/SIL/SILFunction.h

```diff
@@ -454,6 +463,15 @@ class SILFunction
     IsWithoutActuallyEscapingThunk = val;
   }
 
+  // SWIFT_ENABLE_TENSORFLOW
+  bool isTFDeabstracted() const {
+    return TFDeabstracted;
+  }
+
+  void setTFDeabstracted(bool val = true) {
```

Comment:
will any caller ever set `val` to false? if we are not sure and just want to write general code here, that'd be fine (or just use a public member as suggested above). consider leaving a comment accordingly though.